py-polars/polars/expr/expr.py

```diff
@@ -959,7 +959,7 @@ def pipe(
         '''
         return function(self, *args, **kwargs)
 
-    def is_not(self) -> Self:
+    def not_(self) -> Self:
```

Comment:
You need to add this new method in addition to the existing one, and deprecate the old method with `@deprectate_renamed_function`. Also add a `.. deprecated::` admonition to the old method.

You can check `Expr.apply` for an example.